src/main/java/fr/moribus/imageonmap/image/ImageRendererExecutor.java

```diff
@@ -113,6 +124,54 @@ public ImageMap run() throws Throwable
         }, callback);
     }
 
+    public static void update(final URL url, final ImageUtils.ScalingType scaling, final UUID playerUUID, final ImageMap map, final int width, final int height, WorkerCallback<ImageMap> callback) {
+        submitQuery(new WorkerRunnable<ImageMap>()
+        {
+            @Override
+            public ImageMap run() throws Throwable
+            {
+
+                final URLConnection connection = HTTPconnection(url);
+
+                final InputStream stream = connection.getInputStream();
+                final BufferedImage image = ImageIO.read(stream);
+                stream.close();
+
+                if (image == null) throw new IOException(I.t("The given URL is not a valid image"));
+
+                // Limits are in place and the player does NOT have rights to avoid them.
+                checkSizeLimit(playerUUID, image );
```

Comment:
Une idée de nom de méthode ? code dupliqué: ` final URLConnection connection = HTTPconnection(url);

                final InputStream stream = connection.getInputStream();
                final BufferedImage image = ImageIO.read(stream);
                stream.close();

                if (image == null) throw new IOException(I.t("The given URL is not a valid image"));

                // Limits are in place and the player does NOT have rights to avoid them.
                checkSizeLimit(playerUUID, image );
`